src/styles/main.scss

```diff
@@ -0,0 +1,46 @@
+@import "./utils/variables";
+@import "./utils/mixins";
+@import "./utils/extends";
+
+.stopwatch {
+  position: relative;
+  @include circle($stopwatch-size);
+  background-color: blanchedalmond;
+  margin: 0 auto;
+
+  &__axis {
+    @extend %position;
+    @include circle($axis-size);
+    top: 50%;
+    transform: translate(-50%, -50%);
+    background-color: blueviolet;
+  }
+
+  &__s {
+    top: 25%;
+    @include arrow($sArrow-height, $sArrow-width);
+    background-color: #db3892;
+    animation: seconds 60s step-start 0.5s 1 normal backwards;
```

Comment:
make your animation infinite. As it's a second hand - animation-duration should be 1s